ledger_device_sdk/Cargo.toml

```diff
@@ -21,13 +21,11 @@ rand_core = { version = "0.6.3", default_features = false }
 zeroize = { version = "1.6.0", default_features = false }
 numtoa = "0.2.4"
 const-zero = "0.1.1"
-
-[target.'cfg(target_os="nanos")'.dependencies]
-ledger_secure_sdk_sys = {path = "../ledger_secure_sdk_sys", version = "1.4.3"}
-
-[target.'cfg(not(target_os="nanos"))'.dependencies]
-ledger_secure_sdk_sys = {path = "../ledger_secure_sdk_sys", version = "1.4.3", features = ["heap"]}
+ledger_secure_sdk_sys = {path = "../ledger_secure_sdk_sys", version = "1.4.3", default_features = false}
```

Comment:
`heap` feature is not enabled by default in the `ledger_secure_sdk_sys`. It think we can omit `default_features = false` in that dependency declaration. Otherwise, if in the future a new feature is added by default to `ledger_secure_sdk_sys`, it would be disabled by this statement.